datalad/crawler/nodes/annex.py

```diff
@@ -336,7 +336,7 @@ def __init__(self, path=None,
         self._providers = Providers.from_config_files()
         self.yield_non_updated = yield_non_updated
 
-        if (not allow_dirty) and self.repo.dirty:
+        if (not allow_dirty) and self.repo.dirty():
```

Comment:
You won't have less changes to scroll, if I rename it to `is_dirty`. And I will not use that property, you request. I can provide it, if you insist, but I will not go through the code, figuring out, where you might want to see the use of `dirty` and where `is_dirty` is appropriate.
If your actual concern is the scrolling, don't use the f*** phone to review. ;-)
Come on - that one time scrolling in a PR review can't be an argument on how the code should look like.